RELEASE_NOTES.rst

```diff
@@ -21,6 +21,68 @@
 
 .. towncrier release notes start
 
+Airflow 2.9.3 (2024-07-02)
+--------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+Time unit for ``scheduled_duration`` and ``queued_duration`` changed (#37936)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+``scheduled_duration`` and ``queued_duration`` metrics are now emitted in milliseconds instead of seconds.
+
+By convention all statsd metrics should be emitted in milliseconds, this is later expected in e.g. ``prometheus`` statsd-exporter.
+
+
+Support for OpenTelemetry Metrics is no longer "Experimental" (#40286)
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Official support for OpenTelemetry
```

Comment:
I dont think this adds more information than the row above?
Maybe we should say something like "experimintal support for OpenTelemetry was added in ____ since then fixes and improvments were added and now we announce the feature as stable"

Reply:
Thanks @eladkal I have updated it in the RELEASE_NOTES.rst.